### ISSUE TITLE
segfault in `tf.image.crop_and_resize` when `boxes` contains large value

### ISSUE THREAD

**Issue by DNXie** [tensorflow/tensorflow#42129]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 18.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): v2.2.0-rc4-8-g2b96f3662b 2.2.0
- Python version: 3.6.9
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

**Describe the current behavior**
`tf.image.crop_and_resize` segfault when there is a very large value in `boxes`. Can also be reproduced in nightly version

**Describe the expected behavior**
Expect no segfault
**Standalone code to reproduce the issue**

~~~python
import tensorflow as tf
tf.image.crop_and_resize(image=tf.zeros((2,1,1,1)), boxes=[[1.0e+40, 0,0,0]], box_indices=[1], crop_size=[1,1])
~~~

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.

~~~python
Segmentation fault (core dumped)
~~~

**Comment by yongtang** [comment]
Added a PR #42143 for the fix.

**Comment by lvyuqi** [comment]
Does this issue affect the 1.15.5 version?

**Comment by mihaimaruseac** [comment]
Yes, but being low severity and given that patching 1.15 and 2.0 is extremely expensive we no longer patched it.

Recommendation is to update past 2.1

**Comment by mgmm13** [comment]
Hello, is there any chance that vulnerability fix will be applied in versions like 2.3.4?

**Comment by mihaimaruseac** [comment]
Hi. Not in 2.3.4, as the 2.3.x has reached end of life.

It is already included in 2.4.0 and later

**Comment by mgmm13** [comment]
@mihaimaruseac noted on this, thank you for the feedback.
